server/public/_redirects

```diff
@@ -0,0 +1 @@
+/* /index.html 200
```

Comment:
What is this?

Reply:
Ah I forgot to remove that file. It was one of the many things I was trying to get browser routing to work properly on refresh.